### PR TITLE
Validation: Added Alt Text to Images

### PR DESCRIPTION
**Pull request by kturetz**
Added alt text to images.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=646) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=646) before we can accept your contribution.<br/><br/>:x: kturetz [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetDocs?pullRequest=646)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetDocs?pullRequest=646) it.</sub>

File: aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md [hunk 1]
```diff
@@ -30,27 +30,27 @@ At the end of this tutorial, you will have an MVC application with the ASP.NET I
 1. Log in to the [Azure Portal](https://go.microsoft.com/fwlink/?linkid=529715&amp;clcid=0x409).
 2. Click **NEW** at the bottom of the page, and then select **STORE**:
 
-    [![Displays menu with STORE entry selected.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image2.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image1.png)
+    [![Screenshot of Azure Portal menu with STORE entry selected.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image2.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image1.png)
```
Comment (brittmsantos) [inline aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md:33]: Too short; consider rewriting to: "Screenshot of the Azure Portal menu with the Store menu item highlighted with a red rectangle."

File: aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md [hunk 2]
```diff
@@ -30,27 +30,27 @@ At the end of this tutorial, you will have an MVC application with the ASP.NET I
...
 3. In the **Choose and Add-on** wizard, select **ClearDB MySQL Database**, and then click the **Next** arrow at the bottom of the frame:
 
    [Click the following image to expand it. ]
-    [![Displays Choose and Add-on wizard, with ClearDB MySQL Database selected.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image4.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image3.png)
+    [![Screenshot of Choose and Add-on wizard, with Clear D B My S Q L Database selected.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image4.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image3.png)
```
Comment (brittmsantos) [inline aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md:37]: A little too short for recommended specs and a typo; consider: "Screenshot of the Choose an Add-on wizard, with Clear D B My S Q L Database highlighted with a red rectangle."

File: aspnet/identity/overview/getting-started/developing-aspnet-apps-with-windows-azure-active-directory.md [hunk 3]
```diff
@@ -52,40 +52,40 @@ The following steps use [Visual Studio Express 2013 for Web](https://www.microso
...
+    ![Screenshot of Sign in dialog, with fields for organizational account username and password displayed.](developing-aspnet-apps-with-windows-azure-active-directory/_static/image12.png)
 8. After successful sign in, the ASP.NET site will show that you've authenticated by displaying the username in the top right corner of the page.
 
-    ![](developing-aspnet-apps-with-windows-azure-active-directory/_static/image13.png)
+    ![Screenshot of A S P dot NET site with u r l highlighted in address bar and username highlighted at top right.](developing-aspnet-apps-with-windows-azure-active-directory/_static/image13.png)
 
    If you get the error:
    Value cannot be null or empty. Parameter name: linkText
-    ![](developing-aspnet-apps-with-windows-azure-active-directory/_static/image14.png)
+    ![Screenshot of server error that may result on sign-in. Error message is highlighted, reading, Value cannot be null or empty. Parameter name colon linkText.](developing-aspnet-apps-with-windows-azure-active-directory/_static/image14.png)
```
Comment (dereklh7) [inline aspnet/identity/overview/getting-started/developing-aspnet-apps-with-windows-azure-active-directory.md:88]: link Text

File: aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md [hunk 4]
```diff
@@ -150,27 +150,27 @@ Once you have completed the steps in the preceding sections, you should test you
...
+    [![Screenshot of My S Q L Workbench tool dialog. A S P dot NET Identity tables created on the Identity My S Q L Database are highlighted at lower left.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image8.jpg)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image7.jpg)
 6. Inspect the **aspnetusers** table for instance to check for the entries as you register new users.
 
    [Click the following image to expand it. ]
-    [![](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image26.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image25.png)
+    [![Screenshot of a s p net users table, with entries displaying Id, UserName, PasswordHash, SecurityStamp, and Discriminator columns.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image26.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image25.png)
```
Comment (dereklh7) [inline aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md:176]: Screenshot of A S P dot net users table, with entries displayed I D, User Name, Password Hash, Security Stamp, and Discriminator columns (may need to be adjusted for character limit)

File: aspnet/identity/overview/getting-started/developing-aspnet-apps-with-windows-azure-active-directory.md [hunk 5]
```diff
@@ -112,21 +112,21 @@ The following steps will show you how to deploy the application to Azure. In the
...
+    ![Screenshot of A S P dot NET site, with u r l of newly published website in address bar.](developing-aspnet-apps-with-windows-azure-active-directory/_static/image18.png)
 
 <a id="dbg"></a>
 ## Debugging the app
 
 If you get the following error:
  Value cannot be null or empty. Parameter name: linkText
 
-![](developing-aspnet-apps-with-windows-azure-active-directory/_static/image19.png)
+![Screenshot of erver error that may result. Error message is highlighted, reading, Value cannot be null or empty. Parameter name colon linkText.](developing-aspnet-apps-with-windows-azure-active-directory/_static/image19.png)
```
Comment (dereklh7) [inline aspnet/identity/overview/getting-started/developing-aspnet-apps-with-windows-azure-active-directory.md:129]: link Text

File: aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md [hunk 6]
```diff
@@ -30,27 +30,27 @@ At the end of this tutorial, you will have an MVC application with the ASP.NET I
...
 4. Keep the default **Free** plan, change the **NAME** to **IdentityMySQLDatabase**, select the region that is nearest to you, and then click the **Next** arrow at the bottom of the frame:
 
    [Click the following image to expand it. ]
-    [![Illustrates how to change the NAME to IdentityMySQLDatabase and select region.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image6.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image5.png)
+    [![Screenshot of Personalize Add-on dialog, with Free plan highlighted and selected and NAME and REGION selected below. NAME field is filled with Identity My S Q L Database and REGION field is filled with the example West U S.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image6.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image5.png)
```
Comment (brittmsantos) [inline aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md:41]: Too lengthy at 223 characters; consider: "Screenshot of the Personalize Add-on dialog showing the Free plan option and the Name and region fields selected and highlighted with a red rectangle."

File: aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md [hunk 7]
```diff
@@ -30,27 +30,27 @@ At the end of this tutorial, you will have an MVC application with the ASP.NET I
...
 6. After your database has been created, you can manage it from the **ADD-ONS** tab in the management portal. To retrieve the connection information for your database, click **CONNECTION INFO** at the bottom of the page:
 
    [Click the following image to expand it. ]
-    [![Illustrates management portal with ADD-ONS tab highlighted at left and CONNECTION INFO highlighted at bottom.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image10.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image9.png)
+    [![Screenshot of management portal with CREDIT STATUS tab selected in top menu, ADD-ONS tab highlighted at left and CONNECTION INFO highlighted at bottom. Identity My S Q L database name is highlighted at right.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image10.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image9.png)
```
Comment (brittmsantos) [inline aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md:49]: Too long at 203 characters; consider: "Screenshot of the management portal with the Add Ons tab, the Identify My S Q L database item, and the Connection Info button highlighted in red."

File: aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md [hunk 8]
```diff
@@ -30,27 +30,27 @@ At the end of this tutorial, you will have an MVC application with the ASP.NET I
...
 7. Copy the connection string by clicking on the copy button by the **CONNECTIONSTRING** field and save it; you will use this information later in this tutorial for your MVC application:
 
    [Click the following image to expand it. ]
-    [![Illustrates Connection info screen with copy button highlighted by the CONNECTIONSTRING field.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image12.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image11.png)
+    [![Screenshot of Connection info dialog with copy button highlighted at right of CONNECTIONSTRING field.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image12.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image11.png)
```
Comment (brittmsantos) [inline aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md:53]: "at right of Connection String field" (so the screen reader software will read it correctly)

File: aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md [hunk 9]
```diff
@@ -60,15 +60,15 @@ To complete the steps in this section of the tutorial, you will first need to in
 2. Click **New Project** from the **Start** page, or you can click the **File** menu and then **New Project**:
 
    [Click the following image to expand it. ]
-    [![Illustrates Visual Studio Start page, with New Project selected at left.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image2.jpg)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image1.jpg)
+    [![Screenshot of Visual Studio Start page, with New Project selected in menu at left.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image2.jpg)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image1.jpg)
```
Comment (brittmsantos) [inline aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md:63]: Too short for recommended length; consider: "Screenshot of the Visual Studio start page showing the New Project option highlighted with a red rectangle."

File: aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md [hunk 10]
```diff
@@ -80,11 +80,11 @@ The MVC application that was created from the Visual Studio 2013 template contai
...
 3. The **Package Manager Console** will appear in the bottom section of Visual Studio. Type &quot;**Update-Package EntityFramework**&quot; and press Enter:
 
    [Click the following image to expand it. ]
-    [![Illustrates Package Manager Console in bottom section of Visual Studio, with Update-Package EntityFramework instruction displayed on command line.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image20.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image19.png)
+    [![Screenshot of Package Manager Console in bottom section of Visual Studio, with Update-Package EntityFramework instruction displayed on command line.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image20.png)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image19.png)
```
Comment (brittmsantos) [inline aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md:87]: "Update Package Entity Framework"

**Comment by kturetz** [comment]
@Rick-Anderson 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**